packages/cdk8s-cli/src/import/crds-dev.ts

```diff
@@ -0,0 +1,48 @@
+import { ImportCustomResourceDefinition, ManifestObjectDefinition } from './crd';
+
+/**
+ *
+ *              github:crossplane/crossplane@0.14.0
+ *              |--^- |  ^         ^         ^ ^  ^
+ *                 |     |         |         | |  |
+ *  - provider ----+     |         |         | |  |
+ *  - account -----------+         |         | |  |
+ *  - repo ------------------------+         | |  |
+ *  - major ---------------------------------+ |  |
+ *  - minor -----------------------------------+  |
+ *  - patch --------------------------------------+
+ */
+
+/**
+ * Matches a https://doc.crds.dev repo
+ *
+ *  - ManifestObjectDefinition[] if found
+ *  - undefined if not
+ *
+ * @param source
+ */
+export async function importCrdsDevRepoMatch (source: string): Promise<undefined | ManifestObjectDefinition[]> {
```

Comment:
Can you please add a unit test for this function under `test/crds-dev.test.ts`?

Reply:
Done, it's got 100% coverage.